app/views/games/show.html.erb

```diff
@@ -1,22 +1,18 @@
-<div id='chessboard'>
+<div class="col-xs-12 col-sm-8 col-sm-offset-2 col-md-6 col-md-offset-3 no-padding">
 	<%= hidden_field :game_id, '', name: nil, id: nil, data: { game_id: @game.id } %>
```

Comment:
Is this what you mean?
```<div id="chessboard" class="col-xs-12 col-sm-8 col-sm-offset-2 col-md-6 col-md-offset-3">```

I removed the 'chessboard' selector because it wasn't defined. I felt it would be pointless if the view called it even though it didn't add style to the page. If there is way to use Bootstrap classes in the stylesheet then I can use them to define the 'chessboard' selector. I'm not sure how to do that at the moment. I'm looking into it. 

The 'no-padding' class is a class I intent to use multiple times in other areas of the app. Bootstrap adds padding on all their column classes. There are instances when this padding should be removed. It's convenient to just call it from a class when needed. If I don't need it as much as I anticipate then I suppose I can just remove it.

What are your thoughts? Any solutions?